### PR TITLE
Implement `context::erase`

### PR DESCRIPTION
**Pull request by dominiklohmann**
This new operaors allows selective deletion of entries from contexts. It's usage is simple: `context::erase name:string, key=any`, where `name` specifies the name of the context to erase entries from, and `key` specifies the keys to delete.

Erasing entries is currently only support for lookup tables.

### CODE REVIEW

File: web/docs/tql2/operators/context/erase.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+The name of the context to remove entries from.
+
+### `key = any`
+
+The field that represents the enrichment key in the data.
+
+## See Also
+
+[`context::update`](update.md),
```
Comment (mavam) [inline web/docs/tql2/operators/context/erase.md:26]: Trailing comma?

File: web/docs/tql2/operators/context/update.md
Comment (mavam) [inline web/docs/tql2/operators/context/update.md:1]: You basically need to update the See Also of _all_ context operators.

**Review by mavam — APPROVED** [review]
Untested. Tests would be nice.

File: contrib/tenzir-plugins
Comment (dominiklohmann) [inline contrib/tenzir-plugins:1]: I want to wait with merging this until after #4839 is in. One change in this needs to be adapted to make use of the new `iter_series` function.

**Comment by dominiklohmann** [comment]
> Tests would be nice.

I plan on rewriting all of our existing contexts tests to TQL2 in the coming days. There's an existing test for TQL1's `context update --erase` that I will replace with this.

**Pull request merged**